android/app/src/main/res/xml/breezapduservice.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Shouldn't this file be called `clovrlabsapduservice.xml`?

Reply:
It is aidl which allows to communicate between process. I will clean the project later. I prefer to leave this file because  it is connected with the NFC Service. Maybe in the feature we will add the feature.